Example/src/Example.android.tsx

```diff
@@ -112,64 +123,64 @@ export default class IBeaconExample extends Component {
         console.log('beaconDidDisappear', lostBeacon, region);
 
         const { beacons } = this.state;
-        const index = beacons.findIndex(beacon =>
+        const index = beacons.findIndex((beacon) =>
           this._isIdenticalBeacon(lostBeacon, beacon)
         );
         this.setState({
-          beacons: beacons.reduce((result, val, ind) => {
+          beacons: beacons.reduce((result: Array<IBeaconAndroid>, val, ind) => {
```

Comment:
A bit more elegant solution migh tbe to use the Generic of `reduce`, i.e. `beacons: beacons.reduce<Array<IBeaconAndroid>>((result, val, ind) => {`, but it's perfectly fine like this!

Reply:
Thank you! This is a much better solution yes